cypress/e2e/auth.js

```diff
@@ -13,23 +13,30 @@ describe('Auth', () => {
     it('Logout and login', () => {
         cy.get('[data-attr=top-menu-item-logout]').click()
 
-        cy.get('[data-attr=login-email]').type('fake@posthog.com').should('have.value', 'fake@posthog.com').blur()
+        cy.get('[data-attr=login-email]').type('test@posthog.com').should('have.value', 'test@posthog.com').blur()
         cy.get('[data-attr=password]', { timeout: 5000 }).should('be.visible') // Wait for login precheck (note blur above)
 
         cy.get('[data-attr=password]').type('12345678').should('have.value', '12345678')
 
         cy.get('[type=submit]').click()
+        // Login should have succeeded
+        cy.location('pathname').should('eq', '/home')
```

Comment:
Interestingly this test didn't actually work, because it didn't check at all whether login succeeded (and it didn't since the email was wrong)